### PR TITLE
Admin mobile friendly

### PR DESCRIPTION
**Pull request by emilypatsko**
Made the admin sightings list page responsive so the table should be good for mobile devices too :)

### CODE REVIEW

File: ClientApp/src/Styles/constants.scss [hunk 1]
```diff
@@ -1,5 +1,6 @@
 @import url('https://fonts.googleapis.com/css2?family=Luckiest+Guy&family=Raleway:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
 
+
```
Comment (chloejonessoftwire) [inline ClientApp/src/Styles/constants.scss:3]: did you mean to put this here?

File: package.json [hunk 2]
```diff
@@ -7,5 +7,10 @@
     "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (chloejonessoftwire) [inline package.json]: dunno if im completely wrong but are you meant to commit the pack.json and package-lock.json files?

**Review by chloejonessoftwire — COMMENTED** [review]
few tiny comments, i know this is a work in progress anyway!! :)

File: ClientApp/src/Styles/constants.scss (hunk 1 above)
Comment (emilypatsko) [reply ClientApp/src/Styles/constants.scss:3]: yep, it's our imported fonts :)

File: package.json (hunk 2 above)
Comment (emilypatsko) [reply package.json]: not sure! I haven't changed the .gitignore at all, so I would assume if they're committing they're supposed to... I'll check

File: ClientApp/src/Pages/AdminSightingsList/AdminSightingsList.scss [hunk 3]
```diff
@@ -1,3 +1,75 @@
...
+    .hide-mobile {
+        display: none;
+    }
+
+    #admin-table {
+        font-size: 14px;
+    }
+}
+
+@media only screen and (max-width: 480px) {
```
Comment (ChristianMcc) [inline ClientApp/src/Pages/AdminSightingsList/AdminSightingsList.scss]: can we put add variables as constants for our different screen sizes?

**Review by ChristianMcc — COMMENTED** [review]
Looking good, can we put some of the screen size breakpoints in our constants file please

File: ClientApp/src/NavBar/NavBar.scss [hunk 4]
```diff
@@ -80,6 +80,7 @@ li a:hover {
   right: 0;
   color: $light-text;
   z-index: 1001;
+  background-color: transparent !important;
```
Comment (ChristianMcc) [inline ClientApp/src/NavBar/NavBar.scss]: another !important we probably want to get rid of :)

**Review by ChristianMcc — COMMENTED** [review]
Looking good! A cheeky !important that I missed first time round then it's ready I think

File: ClientApp/src/NavBar/NavBar.scss (hunk 4 above)
Comment (emilypatsko) [reply ClientApp/src/NavBar/NavBar.scss]: I had addressed this but either forgot to delete this line or somehow it got added back in! On it

**Review by ChristianMcc — APPROVED** [review]

**Pull request merged**